directives/btn-group/btn-group.tpl.html

```diff
@@ -0,0 +1,3 @@
+<div class="wfm-btn-group" ng-class="{'material-depth-1':$ctrl.btnClass == 'wfm-btn-default'}">
+  <button ng-repeat="item in $ctrl.items" class="wfm-btn {{$ctrl.btnClass}}" ng-class="{'{{$ctrl.selectionClass}}': $ctrl.internalSelect === item, 'first':$first, 'last':$last}" ng-click="$ctrl.output(item); $ctrl.internalSelect = item">{{item}}</button>
```

Comment:
Since `selected` is not going to be modified inside the component, should we still keep track of the selected index in `internalSelect`?

Reply:
sorry could you clarify? 
are you saying we should expose the index using the selected binding? 
or that tracking by index is better in case we have duplicates in the items arr?

right now the internalSelect is only used to apply the class. do you need to know the index if the callback function gives you the selected item?

Reply:
I mean we don't need `internalSelect`. Use `selected` to apply the class. 

Consider a case where `output` does not update `selected`, or delay updating it, `internalSelect` and the selection will be out of sync.

Reply:
I just mean we should use `selected` to apply the class. 

Consider a case where `output` does not update `selected`, or delays updating it, `internalSelect` and the selection will be out of sync. The component should reflect `selected` rather than `internalSelect`